airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -206,6 +212,104 @@ def _get_credentials(self, region_name):
             endpoint_url,
         )
 
+    def _assume_role(
+            self,
+            sts_client,
+            extra_config,
+            role_arn,
+            assume_role_kwargs):
```

Comment:
Can you please add type annotation? Those really help :)

Reply:
Thanks, done